libass/ass_fontselect.c

```diff
@@ -445,21 +440,12 @@ ass_font_provider_add_font(ASS_FontProvider *provider,
     for (j = 0; j < meta->n_fullname; j++)
         printf("'%s' ", meta->fullnames[j]);
     printf("\n");
-    printf("  slant: %d\n", meta->slant);
+    printf("  style_flags: %x\n", (unsigned)meta->style_flags);
```

Comment:
Considering it’s `FT_Long`, `%lx` and `(unsigned long)` or perhaps to make sure even `%llx` and `(unsigned long long)` should be a better fit here

Reply:
`%lx` and no cast should be enough IMO. `FT_Long` is [documented](https://freetype.org/freetype2/docs/reference/ft2-basic_types.html#ft_long) to be `signed long` and doesn’t seem subject to change. For comparison, FreeType also has `FT_Int32` and `FT_Int64`, whose underlying types are unspecified.

Reply:
Done, and also:
- Changed the type of `style_flags` to within `ASS_FontInfo` to `FT_Long`
- Made `fsSelection_to_style_flags` static (it's no longer used outside `ass_font.c`)